recipes/example-new-recipe/recipe.yaml

```
@@ -0,0 +1,99 @@
+# This example shows how to define a recipe using the new YAML based recipe format introduced by 
+# CEP 13.
+
+# For more information about this format see: https://prefix-dev.github.io/rattler-build/latest/reference/recipe_file/
+
+# The main differences with the old format is that no preprocessing is required for the file to be valid YAML.
+# This means:
+# - No "selectors", use YAML if-then-else expressions instead (https://prefix-dev.github.io/rattler-build/latest/selectors/)
+# - Jinja expressions are formatted with `${{}}`
+
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Define variables in this section that you can use in other parts.
+context:
+  name: simplejson
+  version: "3.8.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/${{ version }}/simplejson-${{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v${{ version }}.tar.gz
+  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
```

I think it'd be helpful to suggest a variant that doesn't persist the file
```suggestion
  # sha256 is the preferred checksum -- you can get it for a file with:
  #  `openssl sha256 <file name>`,
  # or, in one go without keeping the file with:
  # `curl -L <url> | sha256
  # You may need the packages or openssl and/or curl, available on conda-forge:
  #  `conda install openssl curl -c conda-forge``
```
However, it should be noted that we're not currently [shipping](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fopenssl-3.3.1-h4bc722e_2.conda) any binaries for `sha256` (or `sha256sum`) in `openssl`, and so whether those get picked up depends entirely what else is on your path. Do you want to open an issue on the openssl feedstock to ship those?